bats/tests/k8s/traefik.bats

```diff
@@ -0,0 +1,75 @@
+# Test case 25 & 26
+
+setup() {
+    load '../helpers/load'
+}
+
+@test 'factory reset' {
+    factory_reset
+}
+
+@test 'start k8s' {
+    start_kubernetes --kubernetes.options.traefik=true
+    wait_for_apiserver
+}
+
+get_host() {
+    if is_windows; then
+        echo "127.0.0.1"
+    else
+        echo "localhost"
+    fi
+}
+
+assert_traefik_pods_are_down() {
+    run kubectl get --all-namespaces pods
+
+    if  [[ "$output" != *"connection refused"* ]] && \
+        [[ "$output" != *"No resources found"* ]] && \
+        [[ "$output" != *"ContainerCreating"* ]] && \
+        [[ "$output" != *"Pending"* ]] && \
+        [[ "$output" != *"Completed"* ]] && \
+        [[ "$output" != *"Terminating"* ]] && \
+        [[ "$output" != *"traefik"* ]]; then
```

Comment:
If the condition is spread out over a long list of continuation lines, then I would prefer to not tack the "then" at the end of the last part of the condition, but have it outdented on the following line:

```suggestion
        [[ "$output" != *"traefik"* ]]
    then
```

That way it becomes clear that "return 0" is part of the then-clause, and not part of the condition.